control/mtrain/doc/DoxygenLayout.xml

```diff
@@ -0,0 +1,194 @@
+<doxygenlayout version="1.0">
```

Comment:
Maybe test to see how we can integrate this into the larger robocup firmware doxygen setup